### PR TITLE
BUGFIX: Fix NULL content type error in ContentElementWrappingService

### PR DESCRIPTION
**Pull request by kitsunet**

### CODE REVIEW

**Review by jonnitto — APPROVED** [review]
Fine by reading

**Review by markusguenther — APPROVED** [review]
Works in my project with that fix.

File: Neos.Neos/Classes/Fusion/ContentElementWrappingImplementation.php [hunk 1]
```diff
@@ -42,7 +42,7 @@ class ContentElementWrappingImplementation extends AbstractFusionObject
      */
     public function getValue()
```
Comment (Sebobo) [inline Neos.Neos/Classes/Fusion/ContentElementWrappingImplementation.php]:
```suggestion
    public function getValue(): string
```

**Review by kdambekalns — APPROVED** [review]
Looks reasonable

**Comment by kdambekalns** [comment]
Q: Is only 8.2 affected?

**Comment by Sebobo** [comment]
Yes I broke it in my wrapping cleanup because I relied on the types in the docblocks

**Review by Sebobo — APPROVED** [review]

**Comment by markusguenther** [comment]
Nice and thanks 💙

**Pull request merged**